engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/ClasspathURLStreamHandlerFactory.scala

```diff
@@ -0,0 +1,14 @@
+package pl.touk.nussknacker.engine.util.config
+
+import java.net.{URL, URLConnection, URLStreamHandler, URLStreamHandlerFactory}
+
+object ClasspathURLStreamHandlerFactory extends URLStreamHandlerFactory {
+  // Change to java.net.spi.URLStreamHandlerProvider when migrated to Java 9+
+  URL.setURLStreamHandlerFactory(this)
```

Comment:
I think this is not safe. This does not give guarantees that it will be called only once per JVM (we can have different classloaders for UI and testing, also in Flink there are many jobs).
What's more, when using in Flink it can lead to classloader leak (URL is in root classloader, while this object lives in job classloader)